ui/app/components/keyboard-shortcuts-modal.js

```diff
@@ -8,6 +9,13 @@ export default class KeyboardShortcutsModalComponent extends Component {
   @service keyboard;
   @service config;
 
+  constructor() {
+    super(...arguments);
+    window.addEventListener('blur', () => {
+      set(this, 'keyboard.displayHints', false);
```

Comment:
`question`: Should this get cleaned up after the component unmounts?

Reply:
I don't think so. displayHints = false is the default / at-rest state.